src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -61,44 +56,37 @@ public Drive(Gyroscope gyroscope) {
     rightEncoder.setDistancePerPulse((6.0 * Math.PI) / 2048.0);
   }
 
-  public void setLeftSpeed(double speed) {
-    this.leftGearboxController.set(speed);
-  }
-
-  public void setRightSpeed(double speed) {
-    this.rightGearboxController.set(speed);
+  private void setLeftSpeed(double speed) {
+    this.leftGearbox.setSpeed(speed);
   }
 
-  public void curvatureDrive(double throttle, double curvature, boolean isQuickTurn) {
-    differentialDrive.curvatureDrive(throttle, curvature, isQuickTurn);
+  private void setRightSpeed(double speed) {
+    this.rightGearbox.setSpeed(speed);
   }
 
-  public void notStraightArcade(double throttle, double turnModifier){
+  public void arcadeDrive(double throttle, double turnModifier){
     differentialDrive.arcadeDrive(throttle, turnModifier);
   }
 
-  public void arcadeDrive(double throttle, double turnModifer) { //already tested, gyro is getting correct angle
-    if(turnModifer == 0.0 && throttle != 0.0) { //says if straight is zero and turn is not zero
+  public void curvatureDrive(double throttle, double curvature) {
+    if(throttle != 0.0 && curvature == 0.0 && !gyroDisabled) { //says if straight is zero and turn is not zero
```

Comment:
Are throttle and curvature already tested for DEADBAND before they get here?

Reply:
Correct, see: https://github.com/PearceRobotics/infinite-recharge-2020/pull/87/files#diff-fcbddf2e1e75740e4f6544945556056aR33

Reply:
Ok.  Could you add a comment to indicate that?